tests/python/frontend/pytorch/test_forward.py

```diff
@@ -3912,6 +3912,106 @@ def forward(self, x):
     verify_model(Flip(axis=-1), input_data=input)
 
 
+@tvm.testing.uses_gpu
+def test_forward_im2col():
+    torch.set_grad_enabled(False)
+
+    class Im2col3x3(Module):
```

Comment:
`Im2col3x3` is not called in verify_script_model?

Reply:
Maybe we don't need two separate modules, but use parameters to pass the kernel_size into one Module, so that we can also support more kernel_size in the unit test. @delldu

Reply:
ONLY for **KISS** in unit test. Please feel free comments ^_^.

Reply:
Modified. Only one class Im2col with kernel, remove script test.